src/app/(nextjs_migration)/(authenticated)/user/layout.tsx

```diff
@@ -27,6 +29,28 @@ const MainLayout = async (props: Props) => {
     return <SignInButton autoSignIn />;
   }
 
+  const accountId = session?.user?.subscriber?.fxa_uid;
+
+  let userId;
+  if (accountId) {
+    // If the user is logged in, use a UUID based on the user's subscriber ID.
+    // Note: we may want to use the FxA UID here, but we need approval for that first.
+    // No need to store in a cookie - this is derived from subscriber ID.
+    if (!process.env.NIMBUS_UUID_NAMESPACE) {
+      throw new Error("env var NIMBUS_UUID_NAMESPACE not set");
+    }
+    userId = uuidv5(accountId.toString(), process.env.NIMBUS_UUID_NAMESPACE);
+  }
+
+  // @see https://github.com/mozilla/experimenter/tree/main/cirrus
+  const serverUrl = process.env.SERVER_URL ?? "http://localhost:6060";
+
+  //@ts-ignore TODO this tells us which features to enable.
```

Comment:
Could you clarify in the comment what error we're suppressing here? It's not clear to me from reading the diff.

Reply:
The error is that `features` is unused - I'll expand on the comment here. It will be used, but for the "A/A" test we don't need it.